### ISSUE TITLE
TopView crashes

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#551]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/551) by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

```
INFO: MSS Version: 1.9.2.
INFO: Python Version: 3.7.6 (default, Jan  8 2020, 19:59:22) 
[GCC 7.3.0]
INFO: Platform: Linux-4.19.0-8-amd64-x86_64-with-debian-10.3 (('64bit', ''))
INFO: Launching user interface...
Traceback (most recent call last):
  File "mss_pyui.py", line 436, in create_new_view
    view_window = topview.MSSTopViewWindow(model=self.active_flight_track)
  File "/home/icg173/src/mss/mslib/msui/topview.py", line 168, in __init__
    self.setupUi(self)
  File "/home/icg173/src/mss/mslib/msui/qt5/ui_topview_window.py", line 21, in setupUi
    self.mpl = MplTopViewWidget(self.centralwidget)
  File "/home/icg173/src/mss/mslib/msui/mpl_qtwidget.py", line 1206, in __init__
    sideview=False, parent=parent, canvas=MplTopViewCanvas())
  File "/home/icg173/src/mss/mslib/msui/mpl_qtwidget.py", line 922, in __init__
    super(MplTopViewCanvas, self).__init__()
  File "/home/icg173/src/mss/mslib/msui/mpl_qtwidget.py", line 67, in __init__
    self.fig = matplotlib.figure.Figure(facecolor="w")  # 0.75
AttributeError: module 'matplotlib' has no attribute 'figure'
```

Does not crash in tests, strangely; maybe due to different order of imports and execution? The missing “figure” is never ever imported anywhere, maybe a strange matplotlib thing.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Added missing import

Fix issue #551

**Comment by ReimarBauer** [comment]
**Original comment by Joern Ungermann (Bitbucket: [joernu76](https://bitbucket.org/joernu76), GitHub: [joernu76](https://github.com/joernu76)).**

----------------------------------------

Fix has been removed.

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Fix issue #551

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in joernu76/mss/issue551 (pull request #838)

Fix issue #551
